modules/gapi/include/opencv2/gapi/cpu/gcpukernel.hpp

```diff
@@ -137,7 +137,8 @@ class GAPI_EXPORTS GCPUKernel
     // This function is a kernel's execution entry point (does the processing work)
     using RunF = std::function<void(GCPUContext &)>;
     // This function is a stateful kernel's setup routine (configures state)
-    using SetupF = std::function<void(const GMetaArgs &, const GArgs &, GArg &)>;
+    using SetupF = std::function<void(const GMetaArgs &, const GArgs &,
+                                      const GCompileArgs &, GArg &)>;
```

Comment:
So now you should understand the point of Context object. :)

Reply:
Yes, got you!